frontend/src/static/js/v_authorship.js

```diff
@@ -277,6 +291,16 @@ window.vAuthorship = {
       this.updateSelectedFiles();
     },
 
+    isValidFile(file) {
+      file.groupLineCnt = this.info.isMergeGroup
+          ? this.getContributionFromAllAuthors(file.authorContributionMap)
```

Comment:
is `getContributionFromAllAuthors` really needed? In `authorContributionMap` (except binary ones), do we put in any author with 0 contributions? If not, why do you just check if there exists any named author other than `-`?

Reply:
![image](https://user-images.githubusercontent.com/54735123/108590327-dc6b2080-739d-11eb-8bdd-36a90b200f23.png)


Thanks for pointing it out. After some trial, I found that marking a file as valid based on whether there exists named author instead of the line count may allow invalid file to sneak in. I am not sure about the exact reason but I assume that it is possible for there to be author with no contribution, which causes this issue.

Reply:
Could you please provide the repo name and the argument you used so that I can have a look into this problem<

Reply:
Actually the issue seems to have been resolved now. Sorry for the inconvenience. I have changed the implementation accordingly.